test/read-only-handler.spec.ts

```diff
@@ -285,10 +285,10 @@ describe('ReadOnlyHandler', () => {
             const proxy = target.getReadOnlyProxy(todos);
             const desc = Object.getOwnPropertyDescriptor(proxy, 'entry');
             const { get } = desc;
-            expect(() => { 
+            expect(() => {
                 get().foo = '';
             }).toThrow();
```

Comment:
The `toThrow` should validate the error message to ensure it's the expected one.